<launch>
    <include file="$(find slaw_bringup)/launch/openni.launch" />
    <node pkg="slaw_object_detection" name="CameraCalibration" 
        type="CameraCalibration2" respawn="false" output="screen" />
</launch>

